backend.py

```diff
@@ -253,8 +256,11 @@ def __init__(self, board, robots, tile_count):
     def __repr__(self):
         return "<State {} {}>".format(self._board, self.robots)
 
-    def as_dict(self):
-        return {"robots": [robot.as_dict() for robot in self.robots]}
+    def as_dict(self, map_name):
+        """
+        Return state as dictionary for sending with server
+        """
+        return {"board": get_map_data(map_name), "robots": [robot.as_dict() for robot in self.robots]}
```

Comment:
Nejsem si jista, jestli je nejlepsi napad importovat zakladni nacitaci funkci z loadingu do backendu - kdyz smysl tvoreni loadingu bylo mit zpracovani map z programu Tiled na jednom miste. Nevim ale, jak to vyresit lip.

Reply:
no nijak jinak to asi nevymyslíme, zatím to tak nechce, zeptám se Petra.

Reply:
@encukou můžeme to tak nechat?

Reply:
Tady je jeste jedna vec: v souboru s mapou uz nejsou policka, takze informace neni kompletni.

Reply:
Tak to mi ale řekněte kde to vezmu, a co tam mam dát prosím. 😂

Reply:
Tenhle PR řeší hlavně předání informací ze serveru klientovi. Otázka jestli ty informace jsou kompletní a použitelné se může dořešit v dalším PR.